modules/accessing-the-administrator-perspective.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// web_console/about-administrator-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-the-administrator-perspective_{context}"]
+= Accessing the Administrator perspective
+
+The *Administrator* perspective in the {product-title} web console provides workflows specific to administrator use cases. You can access the *Administrator* perspective from the web console as follows:
+
+.Prerequisites
+To access the *Administrator* perspective, ensure that you have logged in to the web console and have the administrator role privileges.
```

Comment:
The perspective shows up for all users, not just administrators. Even though it's more admin-oriented, there are things any user can do, and we do our best to hide nav items and actions the user can't access.

Reply:
```suggestion
To access the *Administrator* perspective, ensure that you have logged in to the web console.
```